### ISSUE TITLE
Maintained

### ISSUE THREAD

**Issue by querdenker2k** [seaside1/jrule#123]
Hi @seaside1,
Is and will JRule still be maintained?
It's currently a little bit frustating because nothing happens.
I am thinking about developing on my fork.

**Comment by seaside1** [comment]
Hi!
Yes it is. I put time on it from time to time. Had a lot on my plate (personal life), but as far as I can see it's getting better, so should have more time to spend. Right now I'm migrating my production system to be inline with main and I still have a few rules left. I feel I can't start looking into the new PR:s until I have a working main system.

There are some things in the latest merged PR:s that causes problems (for instance removed color classes and functions) and it takes time to verify that it still works in my system when I move them into my local code.

I think we should really try to keep the PR:s as small as possible and also maintain some backwards compatibility in some cases. You me and @seime  can approve PR:s. So as long as two of us agree we can merge PR:s.

Feel bad making excuses but I need some time catching up.

BR S

**Comment by seime** [comment]
I'm also just a bit behind on everything, but I'll catch up soon. Looking forward to add more features and do less refactoring :)

**Comment by seime** [comment]
I'll close this one.

BTW I added Github discussions where we can discuss stuff without creating issues (that IMHO is for bugs/technical problems/feature requests)
